.github/workflows/build-and-push.yml

```diff
@@ -38,14 +38,6 @@ jobs:
           ver="${{ matrix.version }}"
           echo "::set-output name=SHORT_VER::${ver//./}"
 
-      - name: Login to Quay.io private registry
```

Comment:
Same question as for line 18: does removing this have some effect on building and pushing the Nginx 1.20 container?

Reply:
The same as above